src/components/vote-list.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import axios from "axios";
+
+export default function VoteList({ name, voteCount, id, i, getCandidateList }) {
+  const Vote = () => {
+    axios
+      .put(process.env.API_HOST + "/candidates/" + id + "/vote/", {
+        params: {},
+      })
```

Comment:
```suggestion
  const handleVote = () => {
    axios
      .put(process.env.API_HOST + `/candidates/${id}/vote/`)
```
1. 이벤트 핸들러의 이름을 정의할 때는 앞에 보통 handle을 붙여줍니다.
2. API document 보시면 해당 엔드포인트로 데이터를 보낼 때  body 객체를 보낼 필요가 없음을 확인할 수 있습니다.